mongodb-high-availability/azuredeploy.json

```diff
@@ -1,65 +1,65 @@
-{
+{ 
     "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
     "contentVersion": "1.0.0.0",
     "parameters": {
         "adminUsername": {
             "type": "string",
             "metadata": {
-                "description": "Administrator user name used when provisioning virtual machines (which also becomes a system user administrator in MongoDB)"
+                "Description": "Administrator user name used when provisioning virtual machines (which also becomes a system user administrator in MongoDB)"
             }
         },
         "adminPassword": {
             "type": "securestring",
             "metadata": {
-                "description": "Administrator password used when provisioning virtual machines (which is also a password for the system administrator in MongoDB)"
+                "Description": "Administrator password used when provisioning virtual machines (which is also a password for the system administrator in MongoDB)"
             }
         },
         "storageAccountName": {
             "type": "string",
             "defaultValue": "",
             "metadata": {
-                "description": "Unique namespace for the Storage Account where the Virtual Machine's disks will be placed (this name will be used as a prefix to create one or more storage accounts as per t-shirt size)"
+                "Description": "Unique namespace for the Storage Account where the Virtual Machine's disks will be placed (this name will be used as a prefix to create one or more storage accounts as per t-shirt size)"
             }
         },
         "region": {

```

Comment:
Total nit, but location is better. Now that we can use these templates in Azure Pack as well as Azure, region doesn't make as much sense.